Sources/XCBuildSupport/PIF.swift

```diff
@@ -925,6 +925,7 @@ public enum PIF {
             case CLANG_COVERAGE_MAPPING_LINKER_ARGS
             case MACH_O_TYPE
             case MACOSX_DEPLOYMENT_TARGET
+            case MTL_COMPILER_PATH
```

Comment:
issue: question: out of curiosity, why are we updating the PIF builders used with the `xcode` build system?